aries-site/src/examples/templates/forms/PayExample.js

```diff
@@ -81,14 +82,35 @@ const creditCardMask = [
   },
 ];
 
+const FormFieldRequiredLabel = props => {
```

Comment:
Would the name `RequiredFormField` or `FormFieldRequired` be more accurate? Wondering since this contains the entire FormField, not just the label.

Reply:
I think `RequiredFormField` is a good naming